ui/lib/replication/addon/routes/application.js

```diff
@@ -14,6 +13,23 @@ export default Route.extend(ClusterRoute, {
   store: service(),
   auth: service(),
   router: service(),
+  capabilities: service(),
+
+  async fetchCapabilities() {
+    const enablePath = (type, cluster) => `sys/replication/${type}/${cluster}/enable`;
+    const perms = await this.capabilities.fetchMultiplePaths([
+      enablePath('dr', 'primary'),
+      enablePath('dr', 'primary'),
+      enablePath('performance', 'secondary'),
+      enablePath('performance', 'secondary'),
+    ]);
+    return {
+      canEnablePrimaryDr: perms[enablePath('dr', 'primary')].canUpdate,
```

Comment:
originally I wrote all these variables as `canEnableDrPrimary` or `canEnablePerformancePrimary` but I found it easier to read with the type at the end and the cluster in the middle. `Dr` seemed to get lost in the middle there